src/core/dataflow/src/DataFlowEngine.cpp

```diff
@@ -406,35 +409,26 @@ DataFlowResult *DataFlowEngine::applyCustomizableForwardAnalysis(
         /*
          * Compute IN[i]
          */
-        auto &inSetOfI = df->IN(i);
+        auto &inSetOfI = getInSetOfInst(df, i);
         computeIN(i, predI, inSetOfI, df);
 
         /*
          * Compute OUT[i]
          */
-        auto &outSetOfI = df->OUT(i);
+        auto &outSetOfI = getOutSetOfInst(df, i);
         computeOUT(i, outSetOfI, df);
 
         /*
          * Update the predecessor.
          */
         predI = i;
-
-        /*
-         * Move the iterator.
-         */
-        iter++;
       }
 
       /*
        * Add successors of the current basic block to the working list.
        */
-      for (auto succBB : successors(bb)) {
-        if (worklingListContent[succBB] == true) {
```

Comment:
Why is this check not needed anymore?

Reply:
The reason I deleted the map data structure ` worklingListContent ` are following:
1. I think a map from a bb to a boolean is not a good, which serves  std::set functionality. 
2. Conservative concern. The `worklingListContent` will skip adding bb into working lis if it is inside the map. Deleting it is conservative, which introduces some overhead. But since if the bb is not changed, it will skip the computing, and also don't need to maintain and update the map,  the overhead is not significant, and code is more easier to read. 
3. it seems like a typo (extra `l` after working), but I am not sure if it is intentionally.